randovania/games/prime2/game_data.py

```diff
@@ -126,6 +126,10 @@ def _hash_words() -> list[str]:
             "Why can't I use the Kinetic Orb Cannon in Sanctuary Entrance?",
             "The morph cannon will only be active once the scan post has been scanned and the room has been reloaded.",
         ),
+        (
+            "How do I use the energy controller transports that are unlocked at the end of the vanilla game?",
+            "You must first return the Sanctuary energy and then collect the second item in Main Energy Controller. Note that using the transports does not require Light Suit.",
```

Comment:
I would personally remove the "Note" and just have it say "Controller. Using the transports"

Reply:
Fixed